src/ai-bundle/src/Command/ChatCommand.php

```diff
@@ -0,0 +1,213 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\AI\AiBundle\Command;
+
+use Symfony\AI\Agent\AgentInterface;
+use Symfony\AI\AiBundle\Exception\InvalidArgumentException;
+use Symfony\AI\Platform\Message\Message;
+use Symfony\AI\Platform\Message\MessageBag;
+use Symfony\AI\Platform\Result\TextResult;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Completion\CompletionInput;
+use Symfony\Component\Console\Completion\CompletionSuggestions;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ChoiceQuestion;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\DependencyInjection\ServiceLocator;
+
+/**
+ * @author Oskar Stark <oskarstark@gmail.com>
+ */
+#[AsCommand(
+    name: 'ai:chat',
+    description: 'Chat with an agent',
+)]
+final class ChatCommand extends Command
+{
+    private AgentInterface $agent;
+    private string $agentName;
+
+    /**
+     * @param ServiceLocator<AgentInterface> $agents
+     */
+    public function __construct(
+        private readonly ServiceLocator $agents,
+    ) {
+        parent::__construct();
+    }
+
+    public function complete(CompletionInput $input, CompletionSuggestions $suggestions): void
+    {
+        if ($input->mustSuggestArgumentValuesFor('agent')) {
+            $suggestions->suggestValues($this->getAvailableAgentNames());
+        }
+    }
+
+    protected function configure(): void
+    {
+        $this
+            ->addArgument('agent', InputArgument::OPTIONAL, 'The name of the agent to chat with')
+            ->setHelp(
+                <<<'HELP'
+                The <info>%command.name%</info> command allows you to chat with different agents.
+
+                Usage:
+                  <info>%command.full_name% [<agent_name>]</info>
+
+                Examples:
+                  <info>%command.full_name% wikipedia</info>
+
+                If no agent is specified, you'll be prompted to select one interactively.
+                
+                The chat session is interactive. Type your messages and press Enter to send.
+                Type 'exit' or 'quit' to end the conversation.
+                HELP
+            );
+    }
+
+    protected function interact(InputInterface $input, OutputInterface $output): void
+    {
+        // Skip interaction in non-interactive mode
+        if (!$input->isInteractive()) {
```

Comment:
useless check. The parent class already skips calling the `interact` method entirely.